trinity/protocol/common/handlers.py

```diff
@@ -46,3 +51,16 @@ def get_stats(self) -> Dict[str, str]:
             for exchange
             in self
         }
+
+
+if TYPE_CHECKING:
+    from mypy_extensions import DefaultArg
```

Comment:
@carver why didn't we need to install `mypy_extensions` in order for this to work?

Reply:
IDK for sure, but it seems to come for free with `mypy`. I'm trusting that CI would have caught any problems. Since it doesn't matter in production, it basically only matters in CI and dev machines, I didn't put more thought into it.